src/LEDMatrixDriver.cpp

```diff
@@ -73,13 +73,12 @@ bool LEDMatrixDriver::getPixel(int16_t x, int16_t y) const
 	return *p & (1 << b);
 }
 
-void LEDMatrixDriver::setColumn(int16_t x, uint8_t value)
-{	
+void LEDMatrixDriver::setColumn(int16_t x, bool value)

```

Comment:
I'm not sure my intention was understood.
This sets a column to a specific set of bits.
You can for example set it to 0xFF to have it totally lit or to 0x55 to have only half of the bits set. 
The new code doesn't do what it's supposed to do, it sets the whole column to a single value, right?